composer.json

```diff
@@ -42,7 +42,7 @@
 	},
 	"scripts":{
 		"typo3-cms-scripts": [
-			"typo3cms install:fixfolderstructure"
+			"@php vendor/bin/typo3cms install:fixfolderstructure"
```

Comment:
Why do you add the path to then binary here? Should work without, isn't it?

Reply:
That's correct. Composer only operates on the first "word". If the script is the first word, it will be searched in include path and the composer bin install dir (set through config). 

If you add "@php" as first word, it will be replaced with the php binarie used to invoke composer (and not which is availagle as `php` ) which helps in multi-php installation environments. But than, you have to provide the correct path to the script.

Same that you cannot use `@php` for example with some bash find magic - it will only replaced if it is the first word. (same as `@composer` to invoke composer binary in a seperate process with used php bin and composer bin).